recipe/stream_mode/config/stream_ppo_trainer.yaml

```diff
@@ -0,0 +1,1079 @@
+# Format checks enforced on CI:
+# 1. Comments must appear above each field.
+# 2. There must be a blank line between each field.
+# 3. Inline comments (after a field on the same line) are not allowed.
+# 4. Indentation level is respected for nested fields.
+
+# dataset config
```

Comment:
please inherit from ppo_trainer.yaml similar to recipe/dapo/config/dapo_trainer.yaml